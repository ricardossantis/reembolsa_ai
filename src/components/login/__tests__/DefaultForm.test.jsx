import React from "react";
import renderer from "react-test-renderer";
import DefaultForm from "../default-form";

describe("Render Test", () => {
  it("renders DefaultForm", () => {
    const component = renderer.create(<DefaultForm/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});
