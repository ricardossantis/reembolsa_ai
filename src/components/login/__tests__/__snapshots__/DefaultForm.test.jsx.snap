// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders DefaultForm 1`] = `
<form
  className="ant-form ant-form-horizontal sc-bdnylx gUWxmY"
  onSubmit={[Function]}
/>
`;
